library/core/src/iter/traits/iterator.rs

```diff
@@ -1161,7 +1161,8 @@ pub trait Iterator {
     ///
     /// let a = [0, 1, 2, -3, 4, 5, -6];
     ///
-    /// let iter = a.iter().map_while(|x| u32::try_from(*x).ok());
+    /// let iter = a.iter().map_while(|x| u32::
+    /// try_from(*x).ok());
```

Comment:
nit: This looks like maybe an accidental newline?

Reply:
It is accidental. Thanks for noticing. I've updated the commit.